packages/peregrine/lib/talons/Header/useCartTrigger.js

```diff
@@ -25,7 +25,7 @@ export const useCartTrigger = props => {
     const [fetchCartId] = useMutation(createCartMutation);
     const fetchCartDetails = useAwaitQuery(getCartDetailsQuery);
 
-    const itemCount = data ? data.cart.total_quantity : 0;
+    const itemCount = !loading && data ? data.cart.total_quantity : 0;
```

Comment:
Without `loading` state check the trigger would flash values when changing between cache/network.

Reply:
This was only an issue with retry logic I believe. But I can take it out, not a big deal.